app/Http/Controllers/AccountController.php

```diff
@@ -94,11 +94,13 @@ public function delete()
 
     public function destroy()
     {
-        $user = Auth::user()->id;
-        $user->delete();
+        $userId = Auth::user()->id;
+        $user = User::find($userId);
```

Comment:
Wow, that old code looks very broken. But I'm pretty sure we can just have this be `$user = Auth::user()`?